[Typescript] TouchRipple.d.ts(2,33): error TS7016: Could not find a declaration file for module 'react-transition-group'

<!--- Provide a general summary of the issue in the Title above -->
Got the following error when compiling:
```TouchRipple.d.ts(2,33): error TS7016: Could not find a declaration file for module 'react-transition-group'```, fixed by adding a dev dependency to ```@types/react-transition-group```
<!--
    Thank you very much for contributing to Material-UI by creating an issue! ❤️
    To avoid duplicate issues we ask you to check off the following list.
-->

<!-- Checked checkbox should look like this: [x] -->
- [x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
<!---
    If you're describing a bug, tell us what should happen.
    If you're suggesting a change/improvement, tell us how it should work.
-->

It should compile without the need of other dependencies

## Current Behavior
<!---
    If describing a bug, tell us what happens instead of the expected behavior.
    If suggesting a change/improvement, explain the difference from current behavior.
-->
It fails to compile if ```@types/react-transition-group``` is not installed

## Your Environment
<!--- Include as many relevant details about the environment with which you experienced the bug. -->

| Tech         | Version |
|--------------|---------|
| Material-UI  |  latest beta       |
| React        |       16  |
| browser      |     Chrome 63    |
| etc          |        TS 2.6.2  |

--- a/package.json
+++ b/package.json
@@ -64,6 +64,7 @@
   },
   "dependencies": {
     "@types/jss": "^9.3.0",
+    "@types/react-transition-group": "^2.0.6",
     "babel-runtime": "^6.26.0",
     "brcast": "^3.0.1",
     "classnames": "^2.2.5",
@@ -94,7 +95,6 @@
     "@rosskevin/react-docgen": "^3.0.0-beta9",
     "@types/enzyme": "^3.1.4",
     "@types/react": "16.0.34",
-    "@types/react-transition-group": "^2.0.6",
     "app-module-path": "^2.2.0",
     "argos-cli": "^0.0.9",
     "autosuggest-highlight": "^3.1.1",

